modules/google_secret_manager/variables.tf

```diff
@@ -0,0 +1,9 @@
+variable "project_id" {
+  type        = string
+  description = "The project ID to deploy to. If unset, the provider's default project is used."
```

Comment:
Can you help me understand the need for variablizing the project id? This introduces the potential for side-effects, IIUC.

Reply:
I made a variable because it's required for `resource "google_project_iam_member" "secret_accessor"`.  Is there another way?

Reply:
Sounds good to me. Good work on this!